crates/wasmtime/src/config.rs

```diff
@@ -1980,6 +1997,22 @@ impl Config {
             }
         }
 
+        tunables.collector = if self.features().gc_types() {
```

Comment:
Instead of `self.features()` can you use the `features` that are in scope from above?